Doc/Manual/Doxygen.html

```diff
@@ -0,0 +1,1735 @@
+href="Customization.html#Customization_features">%feature directives</a>:
+</p>
+
+<h4>doxygen:notranslate</h4>
+
+<p>
+Turns off translation of Doxygen comments to the target language syntax: the
+original comment will be copied to the output unchanged. This is useful if you
+want to use Doxygen itself to generate documentation for the target language
+instead of the corresponding language tool (<tt>javadoc</tt>, <tt>sphinx</tt>,
+...).
+</p>
+
+
+<h4>doxygen:alias:&lt;command-name&gt;</h4>
+
+<p>
+Specify an alias for a Doxygen command with the given name. This can be useful
+for custom Doxygen commands which can be defined using <tt>ALIASES</tt> option
+for Doxygen itself but which are unknown to SWIG. <tt>"command-name"</tt> is the
+name of the command in the Doxyfile, e.g. if it contains
+</p>
+
+<div class="code"><pre>
+ALIASES = "sideeffect=\par Side Effects:\n"
+</pre></div>
+
+<p>
+Then you could also the same expansion for SWIG with:
+</p>
+
+<div class="code"><pre>
+%feature("doxygen:alias:sideeffect") "\par Side Effects:\n"
+</pre></div>
+
+<p>
+Please note that command arguments are not currently supported with this
+feature.
+</p>
+
+<p>
+Notice that it is perfectly possible and potentially useful to define the alias
+expansion differently depending on the target language, e.g. with
+</p>
+
+<div class="code"><pre>
+#ifdef SWIGJAVA
+%feature("doxygen:alias:not_for_java") "This functionality is not available for Java"
+#else
+%feature("doxygen:alias:not_for_java") ""
+#endif
+</pre></div>
+
+<p>
+you could use <tt>@not_for_java</tt> in the documentation comments of all
+functions which can't, for whatever reason, be currently exposed in Java
+wrappers of the C++ API.
+</p>
+
+
+<h4>doxygen:ignore:&lt;command-name&gt;</h4>
+
+<p>
+This feature allows to just ignore an unknown Doxygen command, instead of
+replacing it with a predefined text as <tt>doxygen:alias</tt> features allows to
+do. For example, you could use
+</p>
+
+<div class="code"><pre>
+%feature("doxygen:ignore:transferfull");
+</pre></div>
+
+<p>
+if you use a custom Doxygen <tt>transferfull</tt> command to indicate that the
+return value ownership is transferred to the caller, as this information doesn't
+make much sense for the other languages without explicit ownership management.
+</p>
+
+<p>
+Doxygen syntax is rather rich and, in addition to simple commands such as
+<tt>@transferfull</tt>, it is also possible to define commands with arguments.
+As explained in <a href="http://www.stack.nl/~dimitri/doxygen/manual/commands.html">Doxygen documentation</a>,
+the arguments can have a range of a single word, everything until the end of
+line or everything until the end of the next paragraph. Currently, only the "end
+of line" case is supported using the <tt>range="line"</tt> argument of the
+feature directive:
+</p>
+
+<div class="code"><pre>
+// Ignore occurrences of
+//
+//    @compiler-options Some special C++ compiler options.
+//
```

Comment:
Silly question probably, but shouldn't this be `@compileroptions` ?

Reply:
And surely the `%feature` statement should be before the comment.

Reply:
Yes, sorry, there is a typo here, it should be the same in both the comment explaining what `%feature` does and the `%feature` itself.

As for the order: it doesn't matter, this is supposed to be an example showing SWIG input, not the example of a comment that is going to be ignore (this is not even a Doxygen comment, as written...). I realize now that it must be confusing, but I'm not sure how to make it more clear: maybe add an example of a Doxygen comment in C++ code using this custom tag too?